etc/kernelspecs/dask_python_yarn_remote/kernel.json

```diff
@@ -0,0 +1,25 @@
+{
+  "language": "python",
+  "display_name": "Dask - Python (YARN Remote Mode)",
+  "metadata": {
+    "process_proxy": {
+      "class_name": "enterprise_gateway.services.processproxies.yarn.YarnClusterProcessProxy"
+    }
+  },
+  "env": {
+    "DASK_YARN_EXE": "/home/testuser/miniconda/envs/dask-env/bin/dask-yarn",
+    "DASK_OPTS": "--name ${KERNEL_ID:-ERROR__NO__KERNEL_ID} --environment python:///opt/anaconda2/bin/python --temporary-security-credentials --deploy-mode remote",
+    "LAUNCH_OPTS": ""
+  },
```

Comment:
Do you need to submit some YARN specific configuration similar to what we have for [Spark on YARN ](https://raw.githubusercontent.com/jupyter/enterprise_gateway/master/etc/kernelspecs/spark_R_yarn_cluster/kernel.json)?

Reply:
The CLI executable here is the [`dask-yarn` CLI](http://yarn.dask.org/en/latest/cli.html) and is already YARN specific.

Reply:
Let's leave like exe then for now.